data/mysql/schema.sql

```diff
@@ -620,6 +620,7 @@ CREATE TABLE `ezcontentobject` (
   `remote_id` varchar(100) DEFAULT NULL,
   `section_id` int(11) NOT NULL DEFAULT '0',
   `status` int(11) DEFAULT '0',
+  `is_hidden` int(11) NOT NULL DEFAULT '0',
```

Comment:
that's quite a lot of space for 0/1. Please use either `tinyint(1)` or simply `boolean`, which is a synonym for the former.